src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/deathofappellant/DeathOfAppellantAboutToSubmitHandler.java

```diff
@@ -5,26 +5,36 @@
 import static uk.gov.hmcts.reform.sscs.ccd.domain.RequestOutcome.GRANTED;
 import static uk.gov.hmcts.reform.sscs.ccd.presubmit.InterlocReviewState.AWAITING_ADMIN_ACTION;
 
+import java.util.List;
 import java.util.Optional;
 import java.util.Set;
+import java.util.function.Predicate;
 import javax.validation.ConstraintViolation;
 import javax.validation.Validator;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
 import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
 import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
 import uk.gov.hmcts.reform.sscs.ccd.domain.*;
 import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.resendtogaps.ListAssistHearingMessageHelper;
+import uk.gov.hmcts.reform.sscs.util.SscsUtil;
 
 @Service
 @Slf4j
 public class DeathOfAppellantAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
 
     private final Validator validator;
+    private final ListAssistHearingMessageHelper hearingMessageHelper;
+    private boolean isScheduleListingEnabled;
 
-    protected DeathOfAppellantAboutToSubmitHandler(Validator validator) {
+    protected DeathOfAppellantAboutToSubmitHandler(Validator validator, ListAssistHearingMessageHelper
+            hearingMessageHelper, @Value("${feature.snl.enabled}") boolean isScheduleListingEnabled) {
```

Comment:
It would be better to have a more verbose name

Reply:
If you are talking about feature toggle flag name, it would not stay in the code and probably like other feature toggles, be removed later. As such, I think it is a very minor thing we can address if needed post next delivery